src/pages/database-main/DatabaseContent.jsx

```diff
@@ -20,13 +20,31 @@ const sideMenu = sideMenuItems.map((key) => ({
   label: `Sider ${key}`,
 }));
 
+const arrayTemp = ["A", "B", "C"];
```

Comment:
for temporary testing array please add comment to indicate this is for testing, and have the variable outside of functional component.